content/docs/sidebar.json

```diff
@@ -100,8 +100,11 @@
         "slug": "how-to",
         "source": false,
         "children": [
-          "add-output-to-stage",
           "undo-adding-data",
+        {
+          "label": "Add Tracked Data to a Stage",
```

Comment:
is it only about tracked data?

it can become tracked probably after this operation

should we use the slug and label?

Reply:
The doc title is too long for the nav bar "Add Dependencies or Outputs to a Stage" even if we shorten "Deps and Outs", I think.

But Tracked Data = Dependencies and Outputs, right? Technically speaking. The meaning/context is different, but what alternative is there? Maybe "Add Existing Data to a Stage" 💡

Reply:
Actually, "Add Deps or Outs to a Stage" seems to fit:

![image](https://user-images.githubusercontent.com/1477535/99774628-ae8eeb00-2ad3-11eb-8c22-cdd8a84d79de.png)

I've updated it now. Resolving